iridium-core-server/src/main/java/software/iridium/api/ApiExceptionHandler.java

```diff
@@ -30,13 +34,28 @@ public class ApiExceptionHandler {
 
   @ResponseStatus(value = HttpStatus.UNAUTHORIZED)
   @ExceptionHandler(NotAuthorizedException.class)
-  public @ResponseBody ApiResponse handleNotAuthorizedException(final Exception e) {
-    return new ApiResponse(HttpStatus.UNAUTHORIZED.toString(), e.getMessage());
+  public @ResponseBody Object handleNotAuthorizedException(
+      final NotAuthorizedException e, final WebRequest request) {
+    final var acceptHeader = request.getHeader(HttpHeaders.ACCEPT);
+    final var shouldReturnJson =
+        acceptHeader != null && acceptHeader.contains(MediaType.APPLICATION_JSON_VALUE);
```

Comment:
I think we will need to  change the logic to something like below:
```
acceptHeader.contains("*json");
```
We have custom media types  across the API in various places, [here is an example](https://github.com/IridiumIdentity/iridium/blob/f7fa2e566072c0345e6ddffa66797868e22a988e/iridium-java-client/src/main/java/software/iridium/api/authentication/domain/ApplicationResponse.java#L18). What do you think?

Reply:
@joshfischer1108 Maybe you could pull that custom media type (from your referenced example) out to a constants file so that it could be more easily referenced here?

Reply:
I'm not a huge fan of constant files.  Here is an example of the custom media type `application/vnd.iridium.id.application-response.1+json`